src/ui-kit/form-templates/international-phone/sam-extension/extension.component.ts

```diff
@@ -40,7 +44,15 @@ export class SamExtension extends SamFormControl {
    */
    placeholder: string = 'ex: 1234';
   
-
+   /**
+    * is extension required
+    */
+   public allowChars = [
+    '0', '1', '2', '3', '4', '5', '6', '7', '8', '9',
+    'backspace', 'left', 'right', 'tab', 'delete'
+  ];
+  private keys: KeyHelper = new KeyHelper(...this.allowChars);
```

Comment:
Also, if you do the above, you don't need to add a property to the class. You can just spread it inside your KeyHelper instantiation here.